### PR TITLE
CHAT-714 The return to line is deleted when editing a message

### PR DESCRIPTION
**Pull request by trongtt**

### CODE REVIEW

File: application/src/main/webapp/js/chat.js [hunk 1]
```diff
@@ -1777,14 +1777,18 @@ ChatApplication.prototype.createDemoUser = function(fullname, email) {
  * @param msgData the raw text of the message to edit
  */
 ChatApplication.prototype.openEditMessagePopup = function (msgDataId, msgData) {
+  console.log(msgData);
```
Comment (thomasdelhomenie) [inline application/src/main/webapp/js/chat.js]: To be removed

File: application/src/main/webapp/js/chat.js [hunk 2]
```diff
@@ -1777,14 +1777,18 @@ ChatApplication.prototype.createDemoUser = function(fullname, email) {
...
     return;
   }
   if (msgData === null || msgData === undefined || msgData === "") {
     return;
   }
 
-  msgHtml = msgData.replace(eval("/<br>/g"), "\n");
+  msgHtml = msgData.replace(/&amp;/g, "&");
+  msgHtml = msgHtml.replace(/&lt;/g, "<");
+  msgHtml = msgHtml.replace(/&gt;/g, ">");
```
Comment (thomasdelhomenie) [inline application/src/main/webapp/js/chat.js]: Just to be sure to understand, these 3 lines are not directly related to the curent issue, right ?

Comment (trongtt) [inline application/src/main/webapp/js/chat.js]: the breakline is just a case, I'm trying to fix some other HTML entities encoded.

File: application/src/main/webapp/js/chat.js (hunk 1 above)
Comment (trongtt) [inline application/src/main/webapp/js/chat.js]: done
